NAPS2.Lib.WinForms/EtoForms/WinForms/WinFormsListView.cs

```diff
@@ -75,38 +79,49 @@ public WinFormsListView(ListViewBehavior<T> behavior)
 
     private void CustomRenderItem(object? sender, DrawListViewItemEventArgs e)
     {
-        int width, height;
         var image = ImageList.Get(e.Item);
-        if (image.Width > image.Height)
-        {
-            width = ImageSize;
-            height = (int) Math.Round(width * (image.Height / (double) image.Width));
-        }
-        else
+        string? label = null;
+        SizeF textSize = SizeF.Empty;
+        int textOffset = 0;
+        if (_behavior.ShowPageNumbers)
         {
-            if (_behavior.ShowPageNumbers)
-                height = ImageSize - 12;
-            else
-                height = ImageSize;
-            width = (int) Math.Round(height * (image.Width / (double) image.Height));
+            label = (e.ItemIndex + 1).ToString() + " / " + _view.Items.Count.ToString();
```

Comment:
While rebasing the changes to this got overwritten. Will replace it with the string interpolation.